src/Tabs.js

```diff
@@ -1,6 +1,6 @@
-import classNames from 'classnames';
```

Comment:
I'm not sure you'll want fd2148942e73c9b7307871d0badae7cec6039bbe, as it is a change to the internals of react-bootstrap that introduces an additional element in the DOM.

Contrast that to d9cf16131c2b82c25abc07e7da93d102a1e54ee7- which is user-facing (where user is the developer using react-bootstrap) with the goal of nudging the user to use ```<Clearfix>``` rather than thinking directly in terms of bootstrap classes.